build.gradle

```diff
@@ -1410,6 +1410,7 @@ project(':storage') {
     implementation project(':storage:api')
     implementation project(':clients')
     implementation project(':metadata')
+    implementation project(':raft')
```

Comment:
I will remove this dependency once `AbstractMetadataRecordSerde` is moved to `clients` module. I plan to do that in a followup PR if we go with this approach.